update_copyrights

```diff
@@ -26,6 +26,16 @@ import sys
 # [6] = current md5 hash
 ##
 
+##
+# Add new licenses to this list:
+# Beware that GNU GPL v2+ can't be relicensed to anything else (e.g. CC BY-SA 4.0) except by the copyright holder.
```

Comment:
Why is GPLv2+ mentioned specifically? It's the same with any other license besides CC0 type of stuff?

Reply:
That comment was indeed too brief to be accurate.  Right, other than licenses like WTFPL and 0BSD and waivers like CC0 and Unlicense, all licenses forbid relicensing the original work (e.g. "this permission notice shall be included in all copies").  What I meant was that some licenses (e.g. permissive licenses or CC BY-SA 4.0 -> GPL v3-only) allow modifications to be under other licenses.  I'll work on clarifying that.

Reply:
Clarified: "Avoid things like 'GNU GPL v2+;CC BY-SA 4.0', unless you mean to dual license under either GNU GPL v2+ or CC BY-SA 4.0.  GNU GPL v2+ and CC BY-SA 4.0 (e.g.  a GNU GPL v2+ file with CC BY-SA 4.0 modifications) isn't legally possible."  Better?